core/controllers/access_validators.py

```diff
@@ -63,6 +63,11 @@ class ClassroomAccessValidationHandler(
 
     @acl_decorators.open_access
     def get(self) -> None:
+        """Retrieves information about a classroom.
+
+        Raises:
+            PageNotFoundException. The classroom cannot be found.
+        """
         assert self.normalized_request is not None
```

Comment:
@JayVivarekar I think you mean this part [here](https://github.com/oppia/oppia/pull/18216#pullrequestreview-1459287158)